src/app/components/top-items/top-items.component.html

```diff
@@ -1,4 +1,11 @@
 <div>
-  <h1>Top Artists:</h1>
-  <p *ngFor="let item of topArtists.items">{{ item.name }}</p>
+  <div *ngIf="itemType=='artists'">
+    <h1>Top Artists:</h1>
+    <p *ngFor="let item of topItems">{{ item.name }}</p>
```

Comment:
Just displaying name for now, might want to create components for artist and track to display more info